javascript/extractor/test/com/semmle/js/extractor/test/AutoBuildTests.java

```diff
@@ -182,6 +182,7 @@ public void basicTest() throws IOException {
     addFile(true, LGTM_SRC, "package.json");
     addFile(true, LGTM_SRC, ".eslintrc.yml");
     addFile(true, LGTM_SRC, "vendor", "leftpad", "index.js");
+    addFile(true, LGTM_SRC, ".xsaccess");
```

Comment:
The tests are still failing. I suspect this line needs to be removed.

Reply:
The trap file you added shows that the extraction happened.  
The trap tests is another way of testing the extractor.